### DISCUSSION TITLE
Computational cost of logging

### DISCUSSION THREAD

**Discussion by MarcusBecker-GitHub** [TUDelft-DataDrivenControl/OFF#5] [General]
@lejeunemax discovered that the way we are currently writing our log statements can lead to an overhead in computation as strings are evaluated regardless of the fact if they are used or not. Changing the notation could move the evaluation to the logger. Maxime, do you still have the details on that?

**Comment by lejeunemax** [comment]
https://github.com/TUDelft-DataDrivenControl/OFF/issues/6
